web/modules/custom/os2forms_email_handler/src/Plugin/WebformHandler/OS2FormsEmailWebformHandler.php

```diff
@@ -279,6 +279,10 @@ private function sendFileSizeNotification(WebformSubmissionInterface $webform_su
   private function getTotalAttachmentFileSize(WebformSubmissionInterface $webform_submission): int {
     $fileElementIds = $this->getFileIdsFromSubmission($webform_submission);
 
+    if (empty($fileElementIds)) {
+      return 0;
+    }
+
     return array_reduce($fileElementIds, function ($carry, $item) {
       return $carry + (int) File::load($item)->getSize();
     });
```

Comment:
Set initial value (cf. https://www.php.net/manual/en/function.array-reduce.php):

```suggestion
    return array_reduce($fileElementIds, function ($carry, $item) {
      return $carry + (int) File::load($item)->getSize();
    }, 0);
```